media/img/firefox/firstrun-horizon/day-scene/sky.svg

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="2100px" height="1180px" viewBox="0 0 2100 1180" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 42 (36781) - http://www.bohemiancoding.com/sketch -->
+    <title>Sunrise - Cropped (Clean)</title>
```

Comment:
It looks like you can nearly half the size of these SVG's by running them through [SVGOMG](https://jakearchibald.github.io/svgomg/). Please also remove the `<title>` and `<description>` tags that contain hard-coded English text.